similar_dev_search/test.py

```diff
@@ -0,0 +1,18 @@
+import utils
```

Comment:
it should be unittest (or pytest) - so you can launch all test together (in future you will have many files with tests) like `python -m unittest discover .`

Reply:
please add it to get additional point for test